### PR TITLE
Integrate skygear-getstarted module

### LINKED ISSUE #252: Add embed get started page from skygear-getstarted

### PR DESCRIPTION
**Pull request by akiroz**
resolves #252

### CODE REVIEW

File: components/Guide/Guide.js [hunk 1]
```diff
@@ -80,6 +80,15 @@ class Guide extends Component {
     );
   }
 
+  get contentComponent() {
+    const { content } = this.props;
+    if (typeof content === 'string') {
+      return (<Markdown content={content} />);
+    } else {
+      return React.createElement(content.component, content.props);
```
Comment (ben181231) [inline components/Guide/Guide.js]: Better to create the React Component and pass to the Guide.

File: components/Guide/Guide.js [hunk 2]
```diff
@@ -80,6 +80,15 @@ class Guide extends Component {
     );
   }
 
+  get contentComponent() {
+    const { content } = this.props;
+    if (typeof content === 'string') {
```
Comment (ben181231) [inline components/Guide/Guide.js]: This case should be the default case. The logic should be:
if is react component then render it, else render as a markdown component.

**Comment by ben181231** [comment]
By the way, you may use `js` file rather than `jsx` file.

**Comment by akiroz** [comment]
My editor doesn't highlight/indent JSX syntax if the extension is `js`...

**Comment by ben181231** [comment]
Configure it. 😅

**Comment by akiroz** [comment]
but I want to separate plain JS and JSX.....

**Comment by akiroz** [comment]
@ben181231 updated.

File: components/Guide/Guide.js [hunk 3]
```diff
@@ -80,6 +80,15 @@ class Guide extends Component {
     );
   }
 
+  get contentComponent() {
+    const { content } = this.props;
+    if (typeof content === 'object') {
```
Comment (ben181231) [inline components/Guide/Guide.js]: Can we use [React.isValidElement](https://facebook.github.io/react/docs/top-level-api.html#react.isvalidelement) to verify it?

File: components/Guide/Guide.js [hunk 4]
```diff
@@ -178,7 +187,10 @@ class Guide extends Component {
 }
 
 Guide.propTypes = {
-  content: PropTypes.string.isRequired,
+  content: React.PropTypes.oneOfType([
+    PropTypes.string.isRequired,
+    PropTypes.object.isRequired,
```
Comment (ben181231) [inline components/Guide/Guide.js]: Should be `React.PropTypes.element`.

**Comment by akiroz** [comment]
Oh I didn't know I could do that... 
I tried `instanceof React.Component` but failed.

**Comment by akiroz** [comment]
@ben181231 updated.

**Pull request merged**